RNTester/android/app/build.gradle

```diff
@@ -120,6 +120,11 @@ android {
             signingConfig signingConfigs.release
         }
     }
+
+    lintOptions {
+        abortOnError false
```

Comment:
can we rather fix the lint errors? might be masking errors, no?

Reply:
@chirag04 we just added RNTester to this job, so I think it's fine to ignore lint errors *right now* to keep tests green. We should, however, have someone send PR(s) to fix any lint warnings that currently exist in master so that we can re-enable lint errors.

I have a similar issue where ShellCheck has surfaced a lot of issues in our shell scripts, but enabling it would turn tests red right away. In that particular case I plan to chip away at the shell issues until ShellCheck can be enabled fully.

Reply:
I gave it a quick run: https://github.com/facebook/react-native/pull/19743 CI is passing